src/views/Play/Play.tsx

```diff
@@ -180,15 +180,15 @@ export class Play extends React.Component<{}, PlayState> {
 
             if (C.rengo) {
                 rengo.push(C);
-            } else if (C.time_per_move > 0 && C.time_per_move < 3600) {
+            } else if (C.time_per_move > 0 && C.time_per_move < 3600) { // TBD: why aren't we using isLive here?
```

Comment:
likely because it predated isLIve?

Reply:
Makes sense - if we're happy with that, I'll clean next time through